src/twisted/pair/ethernet.py

```diff
@@ -15,14 +15,18 @@
 from zope.interface import implementer, Interface
 
 
+
 class IEthernetProtocol(Interface):
     """An interface for protocols that handle Ethernet frames"""
-    def addProto():
+    def addProto(num, proto):
```

Comment:
this interface change looks good can can be done here... but maybe use a `.remove` release note to let other know that the IEthernetProtocol interface was updated to match all the existing implementations.

Reply:
I added newsfragment